content/docs/studio/get-started.md

```diff
@@ -105,6 +105,13 @@ You should now see that a project has been added in your dashboard.
 
 2. You can also make the project public by [sharing it on the web].
 
+## Run experiments and get live updates
+
+1. You can use your project to [submit new experiments] from Iterative Studio.
```

Comment:
Seems repetitive vs the previous section: we already mentioned "After creating a project, you can... submit new experiments"

Can we combine the sections instead?

Reply:
Have updated, but somewhat differently from your suggestion.

Reply:
Thanks @tapadipti and sorry for the quite-late follow-up. I'll send another PR if there are any important suggestions...